swap_meet/vendor.py

```diff
@@ -1,2 +1,81 @@
+from swap_meet.item import Item
+from swap_meet.clothing import Clothing
+from swap_meet.decor import Decor
+from swap_meet.electronics import Electronics
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        if inventory is None:
+            inventory = []
+        self.inventory = inventory
+               
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+        
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)    
+            return item
+
+
+    def get_by_category(self, category):
+        items = []
+        for item in self.inventory:
+            if item.category == category:
+                items.append(item)
+        return items
+
+    def swap_items(self, another_vendor, my_item, their_item):
+        if my_item not in self.inventory or their_item not in another_vendor.inventory:
+            return False
+        else:
```

Comment:
The else is not necessary! Since the if has a return statement, if we move past it, we know we're in the else. The other statements don't need to be nested under an else.